user-script-register-v2/manifest.json

```diff
@@ -0,0 +1,40 @@
+{
+
+  "manifest_version": 2,
+  "name": "User script",
+  "version": "2.0",
+
+  "applications": {
+    "gecko": {
+      "id": "user-script-example@mozilla.org",
```

Comment:
This has the same ID as "user-script", so if we do want to keep both, it should be changed.